crates/core/src/yarn_fn/function_registry.rs

```diff
@@ -2,6 +2,10 @@ use crate::prelude::*;
 use alloc::borrow::Cow;
 #[cfg(feature = "bevy")]
 use bevy::prelude::*;
+
+#[cfg(not(feature = "std"))]
```

Comment:
Since we depend on hashbrown anyways due to most users building this in combination with Bevy, we can use hashbrown for all maps. 
You can just remove the std variants :)

Reply:
done